scripts/fiber/tests-failing.txt

```diff
@@ -42,8 +42,8 @@ src/renderers/dom/shared/__tests__/ReactDOMTextComponent-test.js
 * can reconcile text arbitrarily split into multiple nodes
 * can reconcile text arbitrarily split into multiple nodes on some substitutions only
 
-src/renderers/dom/shared/__tests__/ReactEventListener-test.js
-* should batch between handlers from different roots
+src/renderers/dom/shared/wrappers/__tests__/ReactDOMInput-test.js
+* should control a value in reentrant events
```

Comment:
Before the updates weren't getting batched so the setState calls had different behavior. The test fails because the props here aren't correct and there is a TODO to fix it:

https://github.com/facebook/react/blob/13cb540bda0a120de87b3cc73230d7b04e0013cc/src/renderers/shared/shared/event/ReactControlledComponent.js#L39

It looks like luck that the test passed before.